integrations/va-auth0/gitbook-manifest.yaml

```diff
@@ -0,0 +1,21 @@
+name: VA-Auth0
+title: VA Auth0
+description: Visitor Authentication with Auth0
+externalLinks:
+    - label: Documentation
+      url: https://www.gitbook.com/integrations/heap
+visibility: public #unlisted
+script: ./src/index.tsx
+scopes:
+    - space:metadata:read
+    - space:visitor:auth
+    - space:content:read
+organization: w45sBUBc1JWE1ktYkQfI #d8f63b60-89ae-11e7-8574-5927d48c4877
```

Comment:
This is the GitBook organization on my local environment. I will update to the gitbook org in prod with visibility set to public when the code changes have been reviewed.

I have this now because I need it to test the integrations locally (according to our development docs)